### ISSUE TITLE
help

### ISSUE THREAD

**Issue by NyxKazuya** [sirmilann/Stealthify-2.0#1]
it is possible to encrypt then compile an exe, with other obfuscators they give an error

**Comment by sirmilann** [comment]
yes. after obfuscation its possible to compile to an exe.

**Comment by NyxKazuya** [comment]
https://i.imgur.com/25MExE7.png

great, run it without compiling to exe it works great, but when compiling to .exe it gives me this error, the module is missing, help me

**Comment by sirmilann** [comment]
tried uninstalling the module and redownloading it?

**Comment by NyxKazuya** [comment]
> ¿Intentó desinstalar el módulo y volver a descargarlo?

Yes, I already did, but the same when compiling the .exe asks me to install the module
